src/main/kotlin/com/launchdarkly/intellij/action/ToggleFlagAction.kt

```diff
@@ -51,31 +51,29 @@ class ToggleFlagAction : AnAction {
      * @param event Event received when the associated menu item is chosen.
      */
     override fun actionPerformed(event: AnActionEvent) {
-        val project = event.project
-        if (project != null) {
-            val selectedNode =
-                project.service<FlagToolWindow>().getPanel().tree.lastSelectedPathComponent as DefaultMutableTreeNode
-            val nodeInfo: FlagNodeParent = selectedNode.userObject as FlagNodeParent
-            // Relies on implicit behavior of key being first child.
-            val flagKey = selectedNode.firstChild.toString().substringAfter(" ")
-            val settings = LaunchDarklyMergedSettings.getInstance(project)
-            val patchComment = PatchComment()
-            val patch = PatchOperation()
-            patch.op = "replace"
-            patch.path = "/environments/" + settings.environment + "/on"
-            patch.value = !nodeInfo.env.on
-            patchComment.patch = listOf(patch)
-            val ldFlag = LaunchDarklyApiClient.flagInstance(project)
-            ApplicationManager.getApplication().executeOnPooledThread {
-                try {
-                    ldFlag.patchFeatureFlag(settings.project, flagKey, patchComment)
-                } catch (e: ApiException) {
-                    val notifier = GeneralNotifier()
+        val project = event.project ?: return
```

Comment:
will this end up as a no-op? I can't think of a scenario where it's possible so it's probably fine, but in the case of no IntelliJ project it would act like it toggled without any change?

Reply:
yea - it shouldn't change the existing behavior, just adds in a return early instead of running in an if block